view/frontend/web/js/view/payment/omise-base-method-renderer.js

```diff
@@ -1,19 +1,63 @@
 define(
     [
+        'jquery',
         'Magento_Checkout/js/model/error-processor',
         'Magento_Checkout/js/model/full-screen-loader',
-        'Magento_Checkout/js/model/url-builder'
+        'Magento_Checkout/js/model/url-builder',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Catalog/js/price-utils'
     ],
     function (
+        $,
         errorProcessor,
         fullScreenLoader,
-        urlBuilder
+        urlBuilder,
+        quote,
+        priceUtils
     ) {
         'use strict';
 
         const RETURN_URL = '/orders/:order_id/omise-offsite';
 
         return {
+            /**
+             * Format Price
+             * 
+             * @param {float} amount - Amount to be formatted
+             * @return {string}
+             */
+            getFormattedAmount: function (amount) {
+                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+            },
+
+            /**
+             * Get formatted message about installment value limitation
+             *
+             * NOTE: this value should be taken directly from capability object when it is fully implemented.
+             *
+             * @return {string}
+             */
+            getMinimumOrderText: function (amount) {
```

Comment:
Is this function and getFormattedAmount needed by ALL renderers?

Reply:
Same for all these added functions

Reply:
yes... we need to add it to all methods, minimum is 20 THB

Reply:
The minimum would change by currency?

Reply:
yes.

Reply:
in thailand is different when you use thb, in jp is different

Reply:
My thoughts were actually heading in the credit card direction. Is the min amount the same for, say, a Thai card and a Singaporean card? Do we even do anything with minimum amounts client side with credit cards or do we let the server check it?

Reply:
I will investigate that in the future, once will be doing minimum order warning in other payment methods.